adafruit_bme680.py

```diff
@@ -324,7 +324,7 @@ class Adafruit_BME680_I2C(Adafruit_BME680):
           will be from the previous reading."""
     def __init__(self, i2c, address=0x77, debug=False, *, refresh_rate=10):
         """Initialize the I2C device at the 'address' given"""
-        import adafruit_bus_device.i2c_device as i2c_device
+        from adafruit_bus_device import i2c_device
```

Comment:
Why are you changing the I2C import here?

Reply:
its kinda the style we use in other drivers - so im not oppsoed to this change :D

Reply:
Hi @brentru. I changed it mostly to make pylint happy.

When I added the SPI code, pylint complained about the SPI import, so I "fixed" it.  Not wanting the library to have two different import styles, the choice was to either change the i2c import to match the new SPI import, which would make pylint happy, or to change the SPI import to match the i2c import. Also, I feel like the imports are a bit cleaner this way.

I put it in it's own commit to make it obvious that it has nothing to do with the